profile/latest/advisor/management/advisor/models.go

```diff
@@ -0,0 +1,40 @@
+package advisor
+
+import (
+	 original "github.com/Azure/azure-sdk-for-go/service/advisor/management/2017-04-19/advisor"
+)
+
+type (
+	 Category = original.Category
```

Comment:
>Category = original.Category [](start = 2, length = 28)

qq, would golint pass? In the alias case, do exported structs need to be include a comment?